src/main/java/io/neonbee/job/RedeployEntitiesJob.java

```diff
@@ -0,0 +1,195 @@
+package io.neonbee.job;
+
+import static io.neonbee.internal.deploy.DeployableVerticle.fromClass;
+import static io.neonbee.internal.deploy.Deployables.fromDeployables;
+import static io.neonbee.internal.scanner.DeployableScanner.scanForDeployableClasses;
+
+import java.time.Duration;
+import java.util.AbstractMap;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import com.google.common.annotations.VisibleForTesting;
+
+import io.neonbee.NeonBee;
+import io.neonbee.NeonBeeDeployable;
+import io.neonbee.NeonBeeProfile;
+import io.neonbee.data.DataContext;
+import io.neonbee.entity.EntityVerticle;
+import io.neonbee.internal.Registry;
+import io.neonbee.internal.cluster.ClusterHelper;
+import io.neonbee.internal.cluster.entity.ClusterEntityRegistry;
+import io.neonbee.internal.deploy.Deployable;
+import io.neonbee.internal.deploy.Deployables;
+import io.neonbee.logging.LoggingFacade;
+import io.vertx.core.Future;
+import io.vertx.core.Verticle;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+/**
+ * A job that redeploys all entity verticles that are not deployed in the cluster.
+ */
+@NeonBeeDeployable(namespace = NeonBeeDeployable.NEONBEE_NAMESPACE, autoDeploy = false)
+public class RedeployEntitiesJob extends JobVerticle {
+
+    private static final LoggingFacade LOGGER = LoggingFacade.create(RedeployEntitiesJob.class);
+
+    private static final Duration DEFAULT_INTERVAL = Duration.ofMinutes(5L);
+
+    /**
+     * Create a new ReregisterEntitiesJob job verticle with the default configuration.
+     */
+    public RedeployEntitiesJob() {
+        this(new JobSchedule(DEFAULT_INTERVAL));
```

Comment:
It would be great if we could make this interval configurable via the verticle config. Also enabling / disabling the verticle.

**Example config (`io.neonbee.job.RedeployEntitiesJob.yaml`):**

```yaml
config:
  enabled: true
  intervalMinutes: 5
```

Reply:
As I understand it, this cannot be realized with the JobVerticle, because the JobSchedule must be passed to the constructor.